main.py

```diff
@@ -1,11 +1,12 @@
 from baselines.common.flags import flags
-from baselines.trainers import BaseTrainer
+from baselines.common.registry import registry
+from baselines.trainers import ActiveDiscoveryTrainer
 
 if __name__ == "__main__":
     parser = flags.get_parser()
     args = parser.parse_args()
 
-    trainer = BaseTrainer(args)
+    trainer = ActiveDiscoveryTrainer(args)
```

Comment:
Could you push `ActiveDiscoveryTrainer` as well please.

We should probably load the trainer specified from a config file instead of hardcoding `ActiveDiscoveryTrainer` in `main.py`.

Reply:
Ah. That was leftover from the beginning of this branch where the active discovery thing was still in scope. I'd prefer to revert back to using `BaseTrainer`. I meant to get rid of all references to `ActiveDiscovery`, but missed that.

Reply:
Reverting to `BaseTrainer` sounds good.